dtfs-central-api/src/types/utilisation-report-status.ts

```diff
@@ -0,0 +1,46 @@
+import { ObjectId } from 'mongodb';
+import { TFMUser } from './users';
+
+export type ReportStatus = 'RECONCILIATION_COMPLETED' | 'REPORT_NOT_RECEIVED';
+
+export type ReportDetails = {
+  month: number;
+  year: number;
+  bankId: string;
+};
+
+type ReportId = {
+  id: string;
+};
+
+type ReportWithStatus = {
+  status: ReportStatus;
+  report: ReportDetails | ReportId;
+};
+
+export type PutReportStatusRequestBody = {
+  user: TFMUser;
+  reportsWithStatus: ReportWithStatus[];
```

Comment:
I didn't know what the best practice was with respect to defining types inside specific `.ts` files (for example, I think each `external-api` file has its own `interfaces` directory rather than a set of shared types). If we are happy to have local types in cases where the type is only needed for that one file then I'm happy to do that